changeset.go

```diff
@@ -75,7 +76,8 @@ func (c *Changeset) GetPreviousColumnValue(column string) (interface{}, bool) {
 
 // ChangesetColumn represents a type and value for a column in a changeset.
 type ChangesetColumn struct {
-	Column string      `json:"column"`
-	Value  interface{} `json:"value"`
-	Type   string      `json:"type"`
+	Column   string          `json:"column"`
+	Value    interface{}     `json:"value"`
+	RawValue json.RawMessage `json:"raw_value"`
```

Comment:
I would suggest at the very least adding a comment on this field so that at least we document that it has a narrow use case and isn't intended to be the place where we normally pull off values for a column. Also, if it's the case that this only applies for JSON, which it appears to be true, then I might suggest renaming this to `JSONRawValue`, so as to avoid any confusion